### DISCUSSION TITLE
Proxmox graphs enablement with docker deployment

### DISCUSSION THREAD

**Discussion by gitcatpeter** [librenms/docker#397] [Q&A]
Hello,
 want to enable full Proxmox monitoring with LibreNMS and following the instruction from here:
https://docs.librenms.org/Extensions/Proxmox/#enabling-proxmox-graphs

I am not quite sure how to translate the following steps to LibreNMS in docker environment:

`cp /opt/librenms-agent/agent-local/proxmox /usr/lib/check_mk_agent/local/proxmox
chmod +x /usr/lib/check_mk_agent/local/proxmox
cp /opt/librenms-agent/check_mk@.service /opt/librenms-agent/check_mk.socket /etc/systemd/system
systemctl daemon-reload
systemctl enable check_mk.socket && systemctl start check_mk.socket`

Thank you.

**Comment by vk6xebec** [comment]
Would like to know the same too.

**Comment by murrant** [comment]
you don't run the agent inside your docker container, you run it on your poxmox host...
